components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -3465,6 +3504,39 @@ subroutine ocn_compute_KPP_input_fields(statePool, forcingPool, &
                       - exp( max(-100.0_RKIND, &
                                  -layerThickness(kmin, iCell)/ &
                            config_flux_attenuation_coefficient_runoff))
+         if (trim(config_subglacial_runoff_mode) == 'data') then
+             if (config_use_sgr_opt_kpp == 1) then
```

Comment:
Would you be open to checking that this is functioning correctly by adapting this test case, polaris's ocean/single_column/cvmix, to that purpose? I think it would work just fine to have SGR flux input at the surface of this non-ice-covered case. Or let me know if you've already done a case that helps verify the KPP implementation.

Reply:
I am not sure what you mean and what that case would reveal, we can discuss that further, also I haven't worked with Polaris yet. I did test the option with config_use_sgr_opt_kpp on and off in compass Ocean0.

Reply:
Ok. Was the comparison between config_use_sgr_opt_kpp on and off in compass Ocean0 useful for verification purposes? If not, I can set up and test the polaris idealized case and explain further how it's useful.

Reply:
There were a few tiny changes in T and S around the plume area, but overall it made little difference whether that option was on or off.